src/hevm/src/EVM/UnitTest.hs

```diff
@@ -670,9 +670,12 @@ formatTestLog events (Log _ args (topic:_)) =
     Just t -> case (Map.lookup (wordValue t) events) of
                    Nothing -> Nothing
                    Just (Event name _ _) -> case name of
-                     "logs" ->
+                     "log" ->
                        Just $ formatSBytes args
 
+                     "logs" ->
+                       Just $ formatSString args
+
```

Comment:
```suggestion
                     "log" ->
                       Just $ formatSString args

                     "logs" ->
                       Just $ formatSSBytes args

```

Reply:
`logs` is now deleted again. Instead we have `log_bytes(bytes)`.